InstallAPIcalls.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+import requests
+import numpy as np
+import time
+import pandas as pd
+import json
+
+def UtilityCheck(utility):
+    valid_utilities = ["speculoos_power"]
+    error_string = "The utility provided is not available within Bayou Energy."
+    if utility not in valid_utilities:
+        return error_string
+    else:
+        return None
+    
+def BayouAPICustomer(utility, email):
+    # Bayou has two environments, staging, as shown below, and production, bayou_domain="bayou.energy"
+
+    bayou_domain = "staging.bayou.energy"
+
+    # Get and manage API keys at f"https://{bayou_domain}/dashboard/keys"
+
+    # API reference: https://docs.bayou.energy/v2.0/reference/authentication 
+    
+    # Note: This is mine (Chad's) personal API key. We may want to hide this during production.
+
+    bayou_api_key = "test_221_f6f1a2d9daef91272215d4761f52ff80465c7dee839b4585a0aab96ddc284159"
+    
+    customer = requests.post(f"https://{bayou_domain}/api/v2/customers", json={
+
+        "utility": utility, # Speculoos is Bayou's fake utility for testing, https://docs.bayou.energy/v2.0/reference/utility-support
+
+        "email" : email # Email address isn't a required field, https://docs.bayou.energy/docs/merge-customer-code-with-your-project
+
+    }, auth=(bayou_api_key, '')).json()
+    
+    return customer
+
+#I made this to test dummy customer data as I've created the dummy customer like 30 times at this point
+def BayouAPICustomer2(utility, email):
+    # Bayou has two environments, staging, as shown below, and production, bayou_domain="bayou.energy"
+
+    bayou_domain = "staging.bayou.energy"
+
+    # Get and manage API keys at f"https://{bayou_domain}/dashboard/keys"
+
+    # API reference: https://docs.bayou.energy/v2.0/reference/authentication 
+    
+    # Note: This is mine (Chad's) personal API key. We may want to hide this during production.
+
+    bayou_api_key = "test_221_f6f1a2d9daef91272215d4761f52ff80465c7dee839b4585a0aab96ddc284159"
```

Comment:
Same here as the previous comment, though I actually missed this LOL. Regardless, test if the api key still works and you or I can change this back